### PR TITLE
various python tweaks for constant slip rate API 

### PR DESCRIPTION
**Pull request by chrisbc**

### CODE REVIEW

File: src/python/automation/build_named_fault_mfd_index.py [hunk 1]
```diff
@@ -42,18 +42,21 @@ def build(self):
                 for filename in fnmatch.filter(files, pattern):
                     folder_path = PurePath(root)
                     if len(folder_path.parts) - len(PurePath(self._dir_name).parts) == 1:
-                        #print(root, filename)
+                        print(root, filename)
```
Comment (voj) [inline src/python/automation/build_named_fault_mfd_index.py:45]: are you checking in some debug code?

File: src/python/automation/run_inversion_diagnostics.py [hunk 2]
```diff
@@ -83,50 +90,48 @@ def run_tasks(general_task_id, solutions):
...
+        print("GENERAL_TASK_ID:", GENERAL_TASK_ID)
+
+        #print('SOLUTIONS', solutions)
+        scripts = []
+        for script_file in run_tasks(GENERAL_TASK_ID, solutions):
+            print('scheduling: ', script_file)
+            scripts.append(script_file)
+
+        print('task count: ', len(scripts))
+        pool.map(call_script, scripts)
```
Comment (voj) [inline src/python/automation/run_inversion_diagnostics.py:131]: should be more efficient to have this outside the `for` loop if there's more than one `inversion_task_id`

**Review by voj — COMMENTED** [review]
looks good, minor comments

File: src/python/automation/build_named_fault_mfd_index.py (hunk 1 above)
Comment (chrisbc) [reply src/python/automation/build_named_fault_mfd_index.py:45]: Yeah,  well the whole thing is on constant debug!. It need some work to move configurations out of scripts and provide some test coverage but I haven't got the time ATM.

**Pull request merged**